enfivy-back/src/main/java/com/envify/back/controller/ScriptGeneratorController.java

```diff
@@ -49,10 +64,35 @@ public StringBuilder getScriptFileString(@RequestBody ScriptRequestBody scriptRe
 		return scriptCode;
 	}	
 	
-	@GetMapping(value = "/file")
-	public @ResponseBody byte[] getFileInbyte() throws IOException {
-	    InputStream in = getClass()
-	      .getResourceAsStream(config.getScriptFilePath());
-	    return IOUtils.toByteArray(in);
-	}
+	@PostMapping("/script/lines")
+	public List<ScriptDto> buildScriptLines(@RequestBody ScriptRequestBodyDto scriptRequestBody) throws EnvifyException {
+		final List<ScriptDto> scripts = new ArrayList<>();
+		final List<String> scriptLabels = new ArrayList<>();
+		final List<String> scriptCommand = new ArrayList<>();
+		
+		// get script path
+		String filePath = scriptGeneratorService.buildFilePath(scriptRequestBody.getConfig());
+		
+		scriptGeneratorService.getScriptCommandAndLabelFromFile(scriptLabels, scriptCommand, filePath);
+		
+		if(scriptCommand.size() == scriptLabels.size()) {
+			scriptGeneratorService.fillScriptsLinesList(scripts, scriptLabels, scriptCommand);
+		} else {
+			throw new EnvifyException("Script Templating error might be considering");
+		}
```

Comment:
Pas be soin de if + else.
dans le if si vérifie si scriptCommand.size() est différent de scriptLabels.size() pour renvoyer une erreur, comme ça tu évites d'avoir un else en utilisant un early return.